metricbeat/module/postgresql/statement/statement.go

```diff
@@ -66,6 +66,10 @@ func (m *MetricSet) Fetch(reporter mb.ReporterV2) error {
 
 	for _, result := range results {
 		data, _ := schema.Apply(result)
+		if _, ok := result["total_time"]; ok {
```

Comment:
Maybe I'll also add a comment here, for a contributor or someone new to the code, to help navigate at first sight.

Reply:
Added.